### DISCUSSION TITLE
Path towards stable release?

### DISCUSSION THREAD

**Discussion by ninadpchaudhari** [Eugeny/tabby#3153] [General]
Just curious, if the developers have an idea of freezing a few features & releasing a stable version instead of having alphas.
Wonderful software :) 

Not that the current release is not stable, but the alpha tag loses it's meaning if we always keep it that way? 
Just curious about the community's thoughts on this.

Just trying to understand the steps/benchmarks when the authors would be ok with the stable release? 
Would be a good list for new users to contribute towards.

**Comment by Eugeny** [comment]
Hi, that was my original plan - release alphas until they are stable enough and then tag it as a release :)
There are still a few outstanding bugs in the SSH plugin that must be solved first, but right now I'm struggling to put in the necessary time to iron them out.

**Reply by ninadpchaudhari** [reply]
Thank you! I understand, maintaining an OS project takes a ton of time! :) 
Always Appreciate your work !! 

Wondering if there is a list of those SSH plugin bugs ( i did not see a page in Wiki or issues labeled with "bugs" )? 
- If there is, I ( or someone in the community ) can try to patch?
- If there is no list, I can start by searching the current issues for bugs in the SSH plugin.

I am looking forward to contributing in any small way I can.

**Reply by Eugeny** [reply]
No separate list, but here's a handful: https://github.com/Eugeny/terminus/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+ssh
